### PR TITLE
Aimfast updates

### PR DESCRIPTION
**Pull request by Athanaseus**
- Update base image to use KERN6
- Add more params
- Update run script

### CODE REVIEW

File: stimela/cargo/cab/aimfast/src/run.py [hunk 1]
```diff
@@ -26,15 +26,21 @@
...
             # Compare models/images in pairs
             if i%2:
                 args += ['{0}{1} {2}'.format(cab['prefix'],
-                                            name, " ".join(compare))]
+                                             name, " ".join(compare))]
                 compare = []
+    elif name in ['compare-online']:
+        compare = []
+        for i, val in enumerate(value):
+            compare.append(val)
```
Comment (o-smirnov) [inline stimela/cargo/cab/aimfast/src/run.py:41]: These three lines seem a very confusing way of saying ``compare = list(enumerate(value))``.

**Review by o-smirnov — APPROVED** [review]

**Comment by o-smirnov** [comment]
ok to test

**Comment by o-smirnov** [comment]
retest this please

**Comment by Athanaseus** [comment]
This also updates base images.  I think it can hold until the next release.

**Comment by SpheMakh** [comment]
Should we merge this before I make a release?

File: stimela/cargo/cab/aimfast/src/run.py (hunk 1 above)
Comment (Athanaseus) [reply stimela/cargo/cab/aimfast/src/run.py]: duly

**Comment by Athanaseus** [comment]
Yes, this should be ready for merge!

**Pull request merged**